src/library-authoring/component-picker/SelectLibrary.scss

```diff
@@ -0,0 +1,3 @@
+.library-list {
+  min-height: 620px;
```

Comment:
Can this style be removed? When the picker is inset in an iframe, this makes the page longer than it needs to be.